decidim-comments/config/locales/ca.yml

```diff
@@ -38,7 +38,7 @@ ca:
           error: S'ha produït un error en votar el comentari.
     components:
       add_comment_form:
-        account_message: <a href="%{sign_in_url}">Inicia sessió</a> o <a href="%{sign_up_url}">crea un compte</a>per afegir el teu comentari.
+        account_message: Inicia sessió o crea un compte per afegir el teu comentari.
```

Comment:
Why changing `ca` locale?

Reply:
There are some tests where the locale is changed to ca and es and this translation is in use. This produces an error because the old translations interpolate 2 values not provided by the changed view